src/Glpi/Plugin/Hooks.php

```diff
@@ -125,7 +125,8 @@ class Hooks
     const PRE_ITEM_LIST           = 'pre_item_list';
     const TIMELINE_ACTIONS        = 'timeline_actions';  // (keys: item, rand)
     const TIMELINE_ANSWER_ACTIONS = 'timeline_answer_actions';  // (keys: item)
-    const SHOW_IN_TIMELINE        = 'show_in_timeline';  // (keys: item)
+    const SHOW_IN_TIMELINE        = 'timeline_items'; // Old constant of hook for BC. Replaced by TIMELINE_ITEMS in GLPI 11.0
```

Comment:
I guess the old constant value should also be kept; plugins may have used it directly.